src/commands/dev/exec.js

```diff
@@ -24,14 +23,12 @@ class ExecCommand extends Command {
     }
 
     const envSettings = await getEnvSettings(site.root)
-    if (envSettings.file) {
-      console.log(
-        `${NETLIFYDEVLOG} Overriding the following env variables with ${chalk.blue(
-          path.relative(site.root, envSettings.file)
-        )} file:`,
-        chalk.yellow(Object.keys(envSettings.vars))
+    if (envSettings.vars.length > 0) {
+      this.log(
+        `${NETLIFYDEVLOG} Adding the following env variables from ${envSettings.files.map(f => chalk.blue(f))}:`,
+        chalk.yellow(envSettings.vars.map(([key]) => key))
       )
-      Object.entries(envSettings.vars).forEach(([key, val]) => (process.env[key] = val))
+      envSettings.vars.forEach(([key, val]) => (process.env[key] = val))
```

Comment:
Instead of calling `Object.entries` on each usage, the function now returns it.

Reply:
We can also avoid `process.env[key] = val` and pass an `env` object to `execa`, but I'll do that in another PR